src/main/java/nextstep/qna/domain/Answers.java

```diff
@@ -0,0 +1,28 @@
+package nextstep.qna.domain;
+
+import nextstep.qna.CannotDeleteException;
+import nextstep.users.domain.NsUser;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Answers {
+    List<Answer> answerList;
```

Comment:
```suggestion
    private List<Answer> answerList;
```
소소한 의견이긴 합니다만 변수명에 형 이름을 잘 사용하진 않습니다 😄 
왜냐하면 자료형이 변경될 경우 변수명도 따라서 변경을 해줘야 하는 고민에 빠지게 되고, 
불필요한 코드 수정이 일어날 수 있기 때문에 왠만하면 지양하는 편입니다. 🙏 

```java
// 만약 List<Answer>가 Set으로 변경되거나 Map으로 변경 된다면 어색해집니다. 
private List<Answer> answerList;

private Set<Answer> answerList; // ??
private Map<String, Answer> answerList; // ??

```
그리고 이미 형이 앞에 있으니 변수명에 또 드러내기 보다는 `answers`나 `answersOfQuestion` 같은 좀 더 
요구사항이 잘 드러나는 네이밍이면 좋을 것 같습니다 🙇

Reply:
자료형이 변경 되면 변수명도 변경을 해줘야 하네요 ... 
반영하겠습니다~! 생각을 못 했던 부분입니다~!  의견 감사합니다